vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
Do you know how this file changed if you never actually ran `trash`? In any case, I don't think it's a big deal if we leave this change in the PR (we can always change it back in follow up PR). I'm mainly curious how this change ended up being included.